modules/template/{{cookiecutter.module_repo_name}}/README.md

```
@@ -107,15 +106,13 @@ cmake --build build -j$(nproc)
 
 Or, without the HoloHub CLI:
 
-{% if cookiecutter.language == 'cpp' %}
-
+{% if cookiecutter.language == 'cpp' -%}
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)